docs/_pages/releases.md

```diff
@@ -13,12 +13,13 @@ sidebar:
 
 * Clarify the date/time type when comparing non-compatible dates and times in `BeEquivalentTo` - [3049](https://github.com/fluentassertions/fluentassertions/pull/3049)
 * Improve the rendering of exception messages when using `WithMessage` for better readability - [3039](https://github.com/fluentassertions/fluentassertions/pull/3039)
+* Added `Should().Throw()`, `ThrowAsync()` and `ThrowWithinAsync()` flavors that don’t require a specific exception type - [3056](https://github.com/fluentassertions/fluentassertions/issues/3056)
```

Comment:
🔧 This needs to become part of a new `8.4.0` section.

Reply:
Additionally we link to the PR resolving an issue and not the issue itself.

Reply:
Yes, exactly.